src/sql/base/browser/ui/checkbox/checkbox.ts

```diff
@@ -16,27 +16,33 @@ export interface ICheckboxOptions {
 	checked?: boolean;
 	onChange?: (val: boolean) => void;
 	ariaLabel?: string;
+	displayCheckmark?: boolean;
```

Comment:
I don't see this being used anywhere. also, what is the purpose of this?

Reply:
You're right. This was left over from changes I had. 

I removed this unused code and moved this code to a new pull request at the request of Chris and Charles:
https://github.com/microsoft/azuredatastudio/pull/16043